mlx/directives/item_matrix_directive.py

```diff
@@ -89,7 +90,9 @@ def perform_replacement(self, app, collection):
         if not source_ids:
             # try to use external targets as source
             for ext_rel in external_relationships:
-                for ext_source, target_ids in collection.get_external_targets(self['source'], ext_rel).items():
+                external_targets = collection.get_external_targets(self['source'], ext_rel)
+                # natural sorting on source
+                for ext_source, target_ids in OrderedDict(natsorted(external_targets.items())).items():
```

Comment:
Can we get this to obey the sorting attribute and to behave same as usual items?

Reply:
The item-matrix directive does not have a sorting attribute. I don't understand what you mean.